quipucords/api/details_report/model.py

```diff
@@ -22,13 +22,3 @@ class DetailsReport(models.Model):
     )
     cached_csv = models.TextField(null=True)
     cached_masked_csv = models.TextField(null=True)
-
-    def __str__(self):
-        """Convert to string."""
-        return (
-            "{"
-            f'"id":{self.id},'
-            ' "report_type":"details", '
-            f' "sources":{self.sources}'
```

Comment:
this is the change that will cause most impact in my evaluation.
I honestly don't think the info is worth for debugging - the tracebacks should be enough IMO


https://github.com/quipucords/quipucords/blob/9de10b9f7613c4a82a203e690b87c4e7eb0d6db7/quipucords/scanner/job.py#L358-L362
https://github.com/quipucords/quipucords/blob/9de10b9f7613c4a82a203e690b87c4e7eb0d6db7/quipucords/scanner/job.py#L374-L379